tools/nl/validator/placetest.py

```diff
@@ -187,18 +191,18 @@ def update_rows(ctx):
         ctx.dcid_names[k] = v[0]
 
     try:
-      url = POP_URL + '&variables=Count_Person&' + '&'.join(
-          'entities=' + e for e in dcids)
+      url = POP_URL + '&variable.dcids=Count_Person&' + '&'.join(
```

Comment:
super nit: since the constant is called "POP_URL" we can put `&variable.dcids=Count_Person` in the constant definition. 
alternatively, we can rename it to OBS_URL and keep this line as is.

otherwise it doesn't really make sense for this to be called POP url if the value doesn't really have population specific params.